updating k8s monitoring charts (#210)
ConsenSys/doc.goquorum: docs/tutorials/kubernetes/deploy-charts.md

        prometheus.io/path: "/debug/metrics/prometheus"
```

Update the admin `username` and `password` in the [monitoring values file](https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/values/monitoring.yml).

> Also remember to configure alerts to the receiver of your choice, that is email, Slack etc and then deploy the chart using:

Reviewer: ```suggestion
Configure alerts to the receiver of your choice (for example, email or Slack), then deploy the chart using:
```
